sQuick.package/MainInterface.class/instance/initialize.st

```diff
@@ -1,8 +1,8 @@
 as yet unclassified
 initialize
 
-	| font |
-	
+	| font resourceFolder |
+	resourceFolder := FileSystem workingDirectory / 'sQuick' / 'resources'.
```

Comment:
I don't know what the best way to handle this is. The problem is that GitFileTree apparently makes you pick a subdirectory for each git repo. Since you're packaging the resources with the config, how will you know which subfolder the user put them into?